@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1719,23 +1719,16 @@ export class TextElementsRenderer {
         renderParams: RenderParams
     ): boolean {
         const poiLabel = labelState.element;
-        const worldPosition = this.m_tmpVector3.copy(poiLabel.position);
-
         const worldOffsetShift = getPropertyValue(
             (poiLabel.poiInfo?.technique as PoiTechnique)?.worldOffset,
             this.m_viewState.env
         );
```

Comment:
This belongs inside the new getWorldPosition function. It's alright if you have to pass the environment, there's other placement functions getting it as parameter already.